WasatchNET/Spectrometer.cs

```diff
@@ -97,6 +97,8 @@ public enum LaserPowerResolution { LASER_POWER_RESOLUTION_100, LASER_POWER_RESOL
         /// <remarks>see Util.wavelengthsToWavenumbers</remarks>
         public double[] wavenumbers { get; protected set; }
 
+        public bool isSPI { get; protected set; } = false;

```

Comment:
When adding public properties to Spectrometer, please add them to ISpectrometer so that COM-based languages (like Delphi, used by Spectragryph etc) can access them.